protocol-definitions/Client.yaml

```diff
@@ -340,6 +352,21 @@ methods:
             since: 2.0
             doc: |
               The partition table. In each entry, it has uuid of the member and list of partitions belonging to that member
+      - name: MemberGroupsView
+        since: 2.8
+        params:
+          - name: version
+            type: int
+            nullable: false
+            since: 2.8
+            doc: |
+              Incremental state version of the partition table
+          - name: memberGroups
+            type: List_List_UUID
+            nullable: false
+            since: 2.8
+            doc: |
+              Grouped members by their UUID. Grouping is done based on PartitionGroupConfig.
```

Comment:
this is only true when RoutingStrategy is PARTITION_GROUPS, it could be different in the future. So maybe we should say according to SubsetRoutingConfig or RoutingStrategy

Reply:
done.